frontend/next.config.js

```diff
@@ -1,6 +1,10 @@
 const nextConfig = {
   images: {
-    domains: ["www.quivr.app", "quivr-cms.s3.eu-west-3.amazonaws.com"]
+    domains: [
+      "www.quivr.app",
+      "quivr-cms.s3.eu-west-3.amazonaws.com",
+      "www.gravatar.com",
```

Comment:
You would also need to add the gravatar url to the ContentSecurityPolicy in `frontend/next.config.js` so it won't be blocked in preview and prod